draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -587,9 +587,10 @@ frame MUST be set to ROQ\_FRAME\_CANCELLED.
 
 When a RoQ sender receives a STOP\_SENDING frame for the last open stream
 available to send RTP/RTCP-data, the RoQ sender MUST open one or more new QUIC
-streams before sending new media frames. Any media frame that has already been
-sent on the QUIC stream that received the STOP\_SENDING frame, MUST NOT be sent
-again on the new QUIC stream(s).
+streams before sending new media frames or continue to send new media frames in
+QUIC datagrams. Any media frame that has already been sent on the QUIC stream
+that received the STOP\_SENDING frame, MUST NOT be sent again on the new QUIC
+stream(s) or datagrams.
```

Comment:
Mentioning datagrams within the section discussing STOP\_SENDING seems confusing, because this is in Section 5.2 (streams), and because datagrams aren't associated with a stream, STOP\_SENDING and RESET\_STREAM won't have any effect on whether the sender continues to send datagrams. 

I'm not sure what you're trying to get at, with this change. I might guess, but I'd be guessing. HELP! :no_mouth:

(Ping me on Slack if we need to talk, please!)